From: Matthew Garrett <mjg@redhat.com>
Date: Wed, 18 Nov 2009 22:02:33 -0500
Subject: [Fedora] [acpi] Disable firmware video brightness change by default
Message-id:
Patchwork-id:
O-Subject:
Bugzilla:

Disable firmware video brightness change on AC/Battery switch by default

Signed-off-by: Aristeu Rozanski <arozansk@redhat.com>

diff --git a/drivers/acpi/video.c b/drivers/acpi/video.c
index 6fa959d..ccb2c00 100644
--- a/drivers/acpi/video.c
+++ b/drivers/acpi/video.c
@@ -2088,7 +2088,7 @@ static int acpi_video_bus_put_devices(struct acpi_video_bus *video)
 
 static int acpi_video_bus_start_devices(struct acpi_video_bus *video)
 {
-	return acpi_video_bus_DOS(video, 0, 0);
+	return acpi_video_bus_DOS(video, 0, 1);
 }
 
 static int acpi_video_bus_stop_devices(struct acpi_video_bus *video)
